website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -19,10 +19,18 @@ Use the [Account API](/cloud-docs/api-docs/account) to get account details, upda
 
 ## Creating an Account
 
-Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
+The recommended method to access Terraform Cloud is with [your HCP account](#continue-with-hcp-account). Alternatively, you may create an account through one of the following methods:
 
 - **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
-- **Sign-Up Page:** For Terraform Cloud, go to [`https://app.terraform.io/signup/account`](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+- **Sign-Up Page:** Creating an account requires a username, an email address, and a password. For Terraform Cloud, go to [`https://app.terraform.io/public/signup/account`](https://app.terraform.io/public/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/public/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+
+### Continue with HCP Account
+
+Accessing Terraform Cloud with [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account) can be done by clicking **Continue with HCP account** button from the **Sign-In** page. 
+
+In the first sign-in, a new Terraform Cloud account will be created and linked to [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account).
+
+If there is an existing and still unlinked Terraform Cloud account with the same email address from [your HCP account](https://cloud.hashicorp.com/docs/hcp/create-account), you'll be able to link those accounts. After this process you won't be able to sign-in using Terraform Cloud's username and password, though you'll be able to unlink your account in **User Settings** page if you choose to.
```

Comment:
```suggestion
### Linking HCP and Terraform Cloud Accounts

You can link an existing Terraform Cloud account to your HCP account so that you can sign in with your HCP credentials and access your existing Terraform Cloud organizations.

The first time you log in with your HCP credentials, Terraform Cloud will search for existing Terraform Cloud accounts with the same email address. If Terraform Cloud finds an unlinked account, it asks you if you want to link that Terraform Cloud account to your HCP account. If there is no existing Terraform Cloud account with that email address, Terraform Cloud creates one and automatically links it to your HCP account. 

Once you link your HCP and Terraform Cloud accounts, you will no longer be able to sign in with your Terraform Cloud username and password.
```

Reply:
Thank you for being so patient! I don't know what happened (this is why I should be more careful when editing through the GitHub UI. It was a bit repetitive. I think this is how I meant everything to read - just one sentence in "Log in" that leads into "linked accounts" :-) 

## Log In With Your HCP Cloud Account

We recommend using a [HashiCorp Cloud Platform (HCP)](https://portal.cloud.hashicorp.com/sign-up) account to log in to Terraform Cloud. Your HCP Account grants access to every HashiCorp product, including HashiCorp Learn and the Terraform Registry. If you use your HCP Account, you will manage user settings like multi-factor authentication and password resets from within HCP instead of the Terraform Cloud UI. 
	
To log in with your HCP account, go to the [Sign In to Terraform Cloud](https://app.terraform.io/) page and click **Continue with HCP account**. Terraform Cloud may ask if you want to link your HCP and Terraform Cloud accounts.

### Linked HCP and Terraform Cloud Accounts

You must link your HCP Account with a Terraform Cloud account before you can log into Terraform Cloud with your HCP credentials. The first time you click **Continue with HCP account**, Terraform Cloud searches for existing Terraform Cloud accounts with the same email address. If there is an unlinked account, Terraform Cloud asks if you want to link it to your HCP account. If there is no existing Terraform Cloud account with that email address, Terraform Cloud creates one and automatically links it to your HCP account. 
	
You can unlink your Terraform Cloud account from your HCP account if your Terraform Cloud account existed before linking. You cannot unlink an autogenerated Terraform Cloud account from your HCP account.
	
To unlink your HCP account from your Terraform Cloud account:
	
	 1. Go to **User settings** and click **HCP Account Linking** in the sidebar. 
	 1. Click ****Unlink account**. Your Terraform Cloud account is no longer linked to your HCP account, and you can begin using your Terraform Cloud credentials to log in. You cannot log in with your HCP account unless you re-link it to your Terraform Cloud account.

Reply:
Thanks @laurapacilio! I agree that we're almost there 😄 

About the label button the correct label is **Continue with HCP account**. We're using an old label in staging. There's a [PR](https://github.com/hashicorp/atlas/pull/12881) in the oven to change this label already.